tools/changelogger/src/render.rs

```diff
@@ -58,6 +58,9 @@ pub struct RenderArgs {
     pub source_to_truncate: PathBuf,
     #[clap(long, action)]
     pub changelog_output: PathBuf,
+    /// where should smity_rs entries marked as server / both be written to
+    #[clap(long, action)]
+    pub server_changelog_output: Option<PathBuf>,
```

Comment:
The other option would be to separate the two under different headings. But I think this would be better and maybe we can also include an emoji icon to indicate if it is a server change or a client.